MapleServer2/Types/Inventory.cs

```diff
@@ -132,15 +152,38 @@ public class Inventory {
             }
         }
 
+        // Checks if item exists already or if slot is taken already.
+        public bool Check(Item item)
+        {
+            if (items.ContainsKey(item.Uid)){
+                return true;
+            }
+            if (GetSlots(item.InventoryType).ContainsKey(item.Slot))
+            {
+                return true;
+            }
+            return false;
+        }
+
+        //For printing items to show amount
+        public void print()
+        {
+            foreach (KeyValuePair<long, Item> i in items)
+            {
+                Console.WriteLine(i.Key);
+                Console.WriteLine(i.Value.Amount);
+            }
+        }
+
         // This REQUIRES item.Slot to be set appropriately
         private void AddInternal(Item item) {
-            Debug.Assert(!items.ContainsKey(item.Uid),
-                "Error adding an item that already exists");
-            items[item.Uid] = item;
+            /*Debug.Assert(!items.ContainsKey(item.Uid),
+                "Error adding an item that already exists");*/
+                items[item.Uid] = item;
```

Comment:
While I'm not super concerned about some of the formatting since we haven't decided on coding standards, we want to try to keep code as clean as possible. You generally only want your code to be tabbed over by one in whatever block its in. The way it was formatted before was fine.

Reply:
This is not resolved, please untab this and the below GetSlot().